vela-templates/addons/terraform/definitions/terraform-azure-storage-account.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: core.oam.dev/v1alpha2
```

Comment:
@maciejgwizdala Could you add your name and contact as top level comment in this file? We would like to give you some credit. Meanwhile in case anyone has issues they can find some help